docs/source/guide/distributed-tools.rst

```diff
@@ -75,3 +75,44 @@ The launch script creates a specified number of training jobs (``--num_trainers`
 In addition, a user needs to specify the number of sampler processes for each trainer
 (``--num_samplers``). The number of sampler processes has to match with the number of worker processes
 specified in :func:`~dgl.distributed.initialize`.
+
+Below shows an example of launching long live servers and multiple distributed training jobs
+in a cluster. ``Long Live Servers`` means :class:`~dgl.distributed.DistGraph` servers are launched
+once only and kept alive even training jobs finish and exit. Then any following training jobs
+could fetch graph structures and data from the same servers. As a result, we do not need to load
+graph multiple times which may take a lot of time for large graph.
```

Comment:
```
It is common that users may want to try different models or training configurations against the same graph data. 
To avoid repetitively loading the same graph data, DGL allows users to launch a persistent graph server to be 
shared across multiple training jobs. A persistent graph server will stay alive even all training workers have 
finished and exited. Below shows an example of launching a persistent graph server:
```

Reply:
accepted.